packages/jest-cli/src/jest.js

```diff
@@ -282,6 +287,7 @@ const runCLI = (
                   ? chalk.dim(' \u203A Press ') + 'u' + chalk.dim(' to update failing snapshots.')
                   : null,
                 chalk.dim(' \u203A Press ') + 'p' + chalk.dim(' to filter by a filename regex pattern.'),
+                chalk.dim(' \u203A Press ') + 'i' + chalk.dim(' to interrupt watch mode.'),
```

Comment:
can we call this `c` for cancel and say `Press c to cancel current test run`? I think that is clearer.

Ideally in the future we'll print this only when tests are actually running, but for now this is fine :)